packages/components-native/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@jobber/components-native",
+  "version": "0.0.0",
+  "license": "MIT",
+  "main": "dist/esm/src/index.js",
+  "module": "dist/esm/src/index.js",
+  "types": "dist/types/src/index.d.ts",
+  "files": [
+    "dist",
+    "src",
+    "!**/__tests__",
+    "!**/__fixtures__",
+    "!**/__mocks__"
+  ],
+  "scripts": {
+    "clean": "rm -rf dist/* tsconfig.tsbuildinfo",
+    "build": "npm run build:clean && npm run compile",
+    "bootstrap": "npm run build",
+    "prepack": "npm run build",
+    "compile": "tsc -b ./tsconfig.esm.json ./tsconfig.types.json",
+    "build:clean": "rm -rf ./dist"
+  },
+  "dependencies": {
+    "@jobber/design": "*",
+    "react-native-svg": "^12.4.4",
+    "ts-xor": "^1.1.0"
+  },
+  "devDependencies": {
+    "@types/node": "*",
+    "@types/react": "*",
+    "@types/react-native": "*",
```

Comment:
Same hesitation as below types that don't match the RN version being used can cause issues. If it would grab it from the parent package.json I would specify a version there

The same applies for React versions here too